src/commandHandlers/bio.ts

```diff
@@ -12,8 +12,8 @@ import { log } from '../logger';
  */
 export const command = async (arg: string, embed: MessageEmbed, message: Message) => {
     const args = arg.split('||');
-    const mention = message.mentions.users.first();
-
+    const mention = message.mentions.users.first()
```

Comment:
I think you accidentally removed semicolon from line 15 😀

Reply:
all good now?

Reply:
Yes, thank you. 😄
I approved the PR 😁

Reply:
yeah acknowledged. Thanks, @alrifay for your help